### ISSUE TITLE
Not sure if Im building correctly because I keep getting errors with it. part 2! 

### ISSUE THREAD

**Issue by brendenderp** [cabaletta/baritone#510]
So last time the admin preemptively destroyed my post so lets remake it until we have something that will help not only me but anyone in the future who runs into the same problem. TLDR of the problem is. opening the file from the gradle doesn't work.  

so step 1! open IntelliJ idea. ok going good. click open project! on the file build.gradle! ok!
![baritone1](https://user-images.githubusercontent.com/29009423/60926610-eb9a6a80-a25b-11e9-85ce-9d11da3c36fe.PNG)

next Im just assuming I click ok because the instructions say nothing about this particular window.
![baritone2](https://user-images.githubusercontent.com/29009423/60926629-ff45d100-a25b-11e9-9d0d-2aea2c9a30c1.PNG)
I went through with overwriting it because apparently it already exists? 
![baritone3](https://user-images.githubusercontent.com/29009423/60926667-1d133600-a25c-11e9-85c2-388f8d0e9bf9.PNG)
I dont want anything I tried in the past to screw with this so I do delete and import
![baritone4](https://user-images.githubusercontent.com/29009423/60926689-31efc980-a25c-11e9-85e4-05c95f88b211.PNG)
OKAY! so now WTF I followed the instructions of opening the project and Im missing key parts!
such as "setupDecompWorkspace" and "genIntellijRuns"
![baritone5](https://user-images.githubusercontent.com/29009423/60926731-4fbd2e80-a25c-11e9-8571-6298ea3674b1.PNG)

if you want you can skip down and comment telling me what I did wrong or you can read through my second attempt!
basically same steps as last time but with out the startup screen
![baritone6](https://user-images.githubusercontent.com/29009423/60926859-b6424c80-a25c-11e9-9b5c-f571965ea990.PNG)
![baritone7](https://user-images.githubusercontent.com/29009423/60926862-b80c1000-a25c-11e9-8a38-6103e013abe6.PNG)
![baritone8](https://user-images.githubusercontent.com/29009423/60926865-b9d5d380-a25c-11e9-8c67-f91064782c87.PNG)
![baritone9](https://user-images.githubusercontent.com/29009423/60926867-bcd0c400-a25c-11e9-8c6f-588db3662493.PNG)
and error of course! . 

most of these problems seem to be stemming from my lack of knowledge of the software but it doesn't help the step by step guide skips steps. Ive taught people how to use c# and if you do something like that you realize that skipping steps in a guide is like giving a half finished book to someone. some people will assume the outcome but those who dont get stuck

**Comment by leijurv** [comment]
`OKAY! so now WTF I followed the instructions of opening the project and Im missing key parts!
such as "setupDecompWorkspace" and "genIntellijRuns"`

This image is taken from SETUP.md

![68747470733a2f2f692e696d6775722e636f6d2f514566567657502e706e67](https://user-images.githubusercontent.com/3837873/60927060-56987100-a25d-11e9-8536-1c44c40bf10c.jpg)

As you can see, to navigate to these tasks you need to expand the Gradle item in the right sidebar, then expand baritone -> baritone -> tasks -> forgegradle.

**Comment by brendenderp** [comment]
thank you for that. but as you can see it appears empty. 
![image](https://user-images.githubusercontent.com/29009423/60927259-1daccc00-a25e-11e9-8113-fcd4822b6e79.png)

**Comment by leijurv** [comment]
Can you expand the two errors in that bottom left pane? What were they

**Comment by brendenderp** [comment]
here they are. 
![image](https://user-images.githubusercontent.com/29009423/60927403-ab88b700-a25e-11e9-9642-2eaecd947641.png)
thank you for helping I was upset before because I felt I was looking for help but couldn't find anyone with the same problem, its with a IDE I've never used and blah blah blah. thanks for your help

**Comment by leijurv** [comment]
cannnnnnnn you actually expand what the errors are please?

**Comment by leijurv** [comment]
wait i can probably tell you what it is

you're trying to build on java 11

**Comment by leijurv** [comment]
downgrade to java 8, there's some internal intellij stuff that you would also need to change

**Comment by brendenderp** [comment]
yes sorry just realized it goes more.
![image](https://user-images.githubusercontent.com/29009423/60927525-1934e300-a25f-11e9-903f-9788e6cf5c17.png)
is my java version effected by what I have installed in IntelliJ or the java installed to my pc. which would be the newest version.

**Comment by brendenderp** [comment]
![image](https://user-images.githubusercontent.com/29009423/60927633-7466d580-a25f-11e9-8b05-333bf7c77fb3.png)

**Comment by brendenderp** [comment]
it appears I have java 8 (this is the only one I have)
![image](https://user-images.githubusercontent.com/29009423/60927769-e3442e80-a25f-11e9-86f9-13180dc2e852.png)

**Comment by leijurv** [comment]
there is some internal intellij setting you'll need to find to change the selected JDK from 11 to 8. that error `package javax.xml.bind does not exist` means that your JDK is too new (they removed this package in the meantime).

**Comment by brendenderp** [comment]
ok! ill look around in intelliJ for that setting. Should I download a different JDK? verrsion 8?

**Comment by leijurv** [comment]
there's a difference between the JDK and the JRE so i would suggest yes, making sure that JDK 8 is installed

**Comment by brendenderp** [comment]
ok! first thank you so much! also second! here's where i'm at. I've got [JDK 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) installed and I believe I selected it in the project structure/project SDK menu. 
![image](https://user-images.githubusercontent.com/29009423/60929439-3620e480-a266-11e9-9c32-e675e6fb74ae.png)
now Ive got the problem of not knowing how to expand the gradle even though its there. double click, expand all? tried them both.
![image](https://user-images.githubusercontent.com/29009423/60929482-610b3880-a266-11e9-84ef-b86fad42bbc4.png)
